<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ASCII-Assistent</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit_Meldungsfenster">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>651</width>
      <height>481</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>821</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_quelle">
       <property name="text">
        <string>Quellverzeichnis:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_quelle"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_quelle">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_ziel">
       <property name="text">
        <string>Zielverzeichniss ascii :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_ziel"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_ziel">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_ziel_ganx">
       <property name="text">
        <string>Zielverzeichniss ganx :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_ziel_ganx"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_ziel_ganx">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_auflisten">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dateinen auflisten</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="autoRepeatInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>169</y>
      <width>166</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_quelldateien_erhalten">
       <property name="text">
        <string>Quelldateinen erhalten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_std_namen">
       <property name="text">
        <string>Standard-Dateinnamen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_ascii">
       <property name="text">
        <string>ascii erzeugen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_ganx">
       <property name="text">
        <string>ganx erzeugen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_drehung">
       <property name="text">
        <string>Drehung Bauteile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_0">
       <property name="text">
        <string>keine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_90">
       <property name="text">
        <string>90 Grad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_180">
       <property name="text">
        <string>180 Grad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_drehung_270">
       <property name="text">
        <string>270 Grad</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_bild">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Bild Ganner</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionWerkzeug_anzeigen"/>
   </widget>
   <addaction name="menuExtras"/>
  </widget>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionWerkzeug_anzeigen">
   <property name="text">
    <string>Werkzeug anzeigen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
